src/intobj.h

```diff
@@ -111,6 +111,25 @@ static inline Int INT_INTOBJ(Obj o)
 #endif
 }
 
+/****************************************************************************
+**
+*F  INT_CAN_BE_INTOBJ( <i> ) . check if a C integer fits in an immediate
+**  integer.
+*/
+static inline Int INT_CAN_BE_INTOBJ(Int i)
```

Comment:
I find this ambiguous -- the name could also be an alternative name for `IS_INTOBJ` (i.e. "this integer could actually be an intobj, as opposed to being a bag ref"). In the description you use "fits", perhaps also use it in the macro name? So e.g. `FITS_INTO_INTOBJ`.